services/src/main/resources/locale/addon/Gamification.properties

```diff
@@ -32,50 +32,86 @@ exoplatform.gamification.gamificationinformation.domain.Teamwork=Teamwork
 exoplatform.gamification.gamificationinformation.domain.Knowledge=Knowledge
 exoplatform.gamification.gamificationinformation.domain.Feedback=Feedback
 exoplatform.gamification.gamificationinformation.myachievements=My Achievements
```

Comment:
no need to change this file. Crowdin will do (see http://developer.exoplatform.org/#id-translations )